gradle.properties

```diff
@@ -1,9 +1,4 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m -Dfile.encoding=UTF-8
 org.gradle.parallel=true
-org.gradle.caching=true
-# AndroidX package structure to make it clearer which packages are bundled with the
-# Android operating system, and which are packaged with the app's APK
-android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
```

Comment:
I think you should still keep it. Picasso uses `android.support` package, so it should be jetified.

https://search.maven.org/artifact/com.squareup.picasso/picasso/2.71828/aar

Reply:
@CherryPerry  Just to be clear, I should keep both the `useAndroidX` and `enableJetifier` flags ?

Reply:
Alright.